### PR TITLE
Moved pycev.py to the root directory

### PR DESCRIPTION
**Pull request by sreekeshiyer**
Referring to Issue #15 

Moved the pycev.py file to the root directory. I've also checked the import statements for the test files, they seem to be okay with this configuration as well.

### CODE REVIEW

**Review by engineerjoe440 — CHANGES_REQUESTED** [review]
Aha! I think all of these buid checks are failing due to a single line in the `setup.py` file. Would you be able to change [line 10](https://github.com/engineerjoe440/pycev/blob/develop/setup.py#L10) as follows:

```diff
# Gather Version Information from Python File
- with open("pycev/pycev.py", encoding="utf-8") as fh:
+ with open("pycev.py", encoding="utf-8") as fh:
    file_str = fh.read()
```

**Comment by engineerjoe440** [comment]
Thank you so much for your efforts, here! 🎉 

They're greatly appreciated, and I think it'll make the overall package much simpler. 😄

**Comment by sreekeshiyer** [comment]
Updated.
I also appreciate the opportunity. I do have quite a bit of experience in development but I'm very new to Open Source Contributions. I've made the changes, I hope it fixes the issue.

File: setup.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/engineerjoe440/pycev",
-    packages=['pycev'],
+    packages=["pycev"],
```
Comment (engineerjoe440) [inline setup.py]: Drat!

I was mistaken in my memory, and it looks like this needs to change as follows:

```diff
    url="https://github.com/engineerjoe440/pycev",
-   packages=["pycev"],
+   py_modules=["pycev"],
    classifiers=[
```

See [example here](https://gist.github.com/arne-cl/c7d156b48a8ecf27f207) for reference.

**Review by engineerjoe440 — CHANGES_REQUESTED** [review]
Lastly, I remembered that there's a simple import test that this will invalidate. That's fine, we just need to remove the test [defined here](https://github.com/engineerjoe440/pycev/blob/develop/tests/unit/test_importpycev.py#L15-L17).

Should be able to just remove it, altogether, and we'll be set!

File: setup.py (hunk 1 above)
Comment (sreekeshiyer) [reply setup.py]: Got it

**Comment by engineerjoe440** [comment]
Thank you, again! I really should have remembered these issues, but I really appreciate your quick response, and willingness to contribute! :tada:

**Comment by engineerjoe440** [comment]
Well...

Good enough for me! 😝 Looks like there's still a few "annoying" little issues in some of my build system scripts (guess I went a little crazy with my package-structure assumptions, before). But I can knock those out pretty easily. I'm not going to ask you to make those dumb little changes.

Thank you again! 🎁 🎉 🎆

**Pull request merged**